scripts/import/import_clinvar_xml

```diff
@@ -1195,10 +1249,19 @@ sub clean_old_data{
 
   my $phenfeat_del_sth   = $dba->dbc->do(qq[ delete from phenotype_feature where source_id in (select source_id from source where name ='ClinVar') ]);
 
-  my $synonym_del_sth    = $dba->dbc->do(qq[ delete from variation_synonym where source_id in (select source_id from source where name ='ClinVar') ]);
-
-  my $var_updt_sth       = $dba->dbc->do(qq[ update variation set clinical_significance=\\N ]);
+  # remove previously inserted variation from ClinVar + associated records
+  my @tables = qw/variation_synonym failed_variation variation_set_variation/;
+  for my $table (@tables){
+    my $var_recs_del_sth    = $dba->dbc->do(qq[ delete from $table where variation_id in
+                                                    (select variation_id from variation v, source s
+                                                     where v.source_id = s.source_id and s.name ='ClinVar')
+                                                    ]);
+  }
+  my $var_feat_del_sth    = $dba->dbc->do(qq[ delete from variation_feature where source_id in (select source_id from source where name ='ClinVar'); ]);
+  my $var_del_sth        = $dba->dbc->do(qq[ delete from variation where source_id in (select source_id from source where name ='ClinVar'); ]);
+  my $synonym_del_sth    = $dba->dbc->do(qq[ delete from variation_synonym where source_id in (select source_id from source where name ='OMIM' ) ]);
 
+  my $var_updt_sth       = $dba->dbc->do(qq[ update variation set clinical_significance=NULL ]);
```

Comment:
This risks the clinical significance info in variation and variation_feature becoming out of sync